CONTRIBUTING.md

```diff
@@ -106,7 +106,11 @@ To make changes to any of the [repository submodules](https://github.com/codacy/
 
 1.  Make the changes directly in the submodule repository.
 
-1.  Back in this repository, create a topic branch.
+1.  Back in this repository, create and checkout a feature branch named `feat/update-<submodule>`. For example:
```

Comment:
I think you are missing one blank space in the indentation of the code block... The preview is not showing correctly.

Reply:
You're right! :sweat_smile: I just fixed it.